torch/utils/data/datapipes/utils/common.py

```diff
@@ -67,12 +68,22 @@ def validate_pathname_binary_tuple(data: Tuple[str, IOBase]):
             f"its subclasses as type, but it is type {type(data[1])}"
         )
 
+
+def deprecation_warning(name, new_name: str = ""):
```

Comment:
Why not we combine these two deprecation warnings into a single function?

Reply:
The deprecation warning for TorchData can be removed in the subsequent PR in the stack because all of those DataPipes will be deleted.

Reply:
Sounds great.